launch.bash

```diff
@@ -154,10 +157,34 @@ function run_test() {
     echo ">>>> End testing."
 }
 
+function run() {
+
+    # change permissions of SSL config files
+    sudo chown -R 999:root configs/
+    sudo chmod 777 configs/
+    sudo chmod 600 configs/*
+
+    # if the image doesn't exist (or we've just deleted it), build it fresh
+    sudo docker image inspect flask-server:v1 >/dev/null 2>&1
+    [[ $? != 0 ]] && echo "Image does not exist; building image" && sudo docker build -t flask-server:v1 .
+
+    # launch and check for 'port in use' error
+    sudo docker-compose up | tee /tmp/out 2>&1
+    if [[ -n $(cat /tmp/out | grep "address already in use") ]]; then
+        sudo service postgresql stop
+        run
+    fi
```

Comment:
I'm not actually sure if this will work, since I don't know how to deterministically recreate this error so I haven't been able to test it. Something to keep an eye on going forward